shared/desktop/webpack.config.base.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
       {
         test: /\.jsx?$/,
         loader: 'babel',
-        exclude: /(node_modules|\/dist\/)/,
+        exclude: /((node_modules\/(?!universalify|fs-extra))|\/dist\/)/,
```

Comment:
ok so now (against best practices) some node_modules are starting to export themselves as es6, which sucks cause we don't transpile node_modules. Now we have to cherry pick a couple of things to transpile